README.md

```diff
@@ -1,28 +1,46 @@
-# Ember Welcome Page
+Ember Welcome Page
+==============================================================================
+
 [![Build Status](https://travis-ci.org/ember-cli/ember-welcome-page.svg?branch=master)](https://travis-ci.org/ember-cli/ember-welcome-page)
 
 Displays a nice welcome page for newly generated applications with links to resources.
 
-## Installation
+Installation
+------------------------------------------------------------------------------
+
+```
+ember install ember-welcome-page
+```
+
+Contributing
+------------------------------------------------------------------------------
+
+### Installation
 
-* `git clone <repository-url>` this repository
+* `git clone https://github.com/ember-cli/ember-welcome-page.git`
 * `cd ember-welcome-page`
-* `npm install`
-* `bower install`
+* `yarn`
 
-## Running
+### Linting
 
-* `ember serve`
-* Visit your app at [http://localhost:4200](http://localhost:4200).
+* `yarn lint:hbs`
+* `yarn lint:js`
+* `yarn lint:js -- --fix`
```

Comment:
I believe this should be `yarn lint:js --fix` (The double dash separating the arg is only needed for npm).